www/js/splash/startprefs.js

```diff
@@ -179,28 +151,6 @@ angular.module('emission.splash.startprefs', ['emission.plugin.logger',
         });
     }
 
-    startprefs.getNextState = function() {
-      return startprefs.getPendingOnboardingState().then(function(result){
-        if (result == null) {
-          if (angular.isDefined($rootScope.redirectTo)) {
-            var redirState = $rootScope.redirectTo;
-            var redirParams = $rootScope.redirectParams;
-            $rootScope.redirectTo = undefined;
-            $rootScope.redirectParams = undefined;
-            return {state: redirState, params: redirParams};
```

Comment:
Needs to be reconsidered when we rewrite `localnotify`. There's no good way to do it right now with `localnotify` still being Angular and using `$rootScope`

For right now, if there's an issue with permissions it doesn't matter whether you open the app and end up on Label or Profile, because the AppStatusModal isn't specific to Profile anymore